app/stacks/uk-west/back-office/service-bus.tf

```diff
@@ -1,9 +1,10 @@
 resource "azurerm_servicebus_namespace" "back_office" {
-  name                = "pins-sb-${local.service_name}-${local.resource_suffix}"
-  location            = azurerm_resource_group.back_office_stack.location
-  resource_group_name = azurerm_resource_group.back_office_stack.name
-  sku                 = var.service_bus_failover_enabled ? "Premium" : "Standard"
-  capacity            = var.service_bus_failover_enabled ? 1 : 0
+  name                          = "pins-sb-${local.service_name}-${local.resource_suffix}"
+  location                      = azurerm_resource_group.back_office_stack.location
+  resource_group_name           = azurerm_resource_group.back_office_stack.name
+  sku                           = var.service_bus_failover_enabled ? "Premium" : "Standard"
+  capacity                      = var.service_bus_failover_enabled ? 1 : 0
+  public_network_access_enabled = false
```

Comment:
So we should actually set this based on service_bus_failover_enabled - we don't have a premium namespace in dev, so we need public network access to be permitted.